usr/sbin/p11sak/p11sak.c

```diff
@@ -960,7 +954,7 @@ static const struct p11sak_opt p11sak_generic_opts[] = {
                      "then you will be prompted for the PIN. If the '--so' "   \
                      "option is specified, specify the SO pin, or supply "    \
                      "the SO pin via environment variable PKCS11_SO_PIN.", },  \
-    { .short_opt = FORCE_PIN_PROMPT_SHORTOPT, .long_opt = "force-pin-prompt", .required = false,       \
+    { .short_opt = 0, .long_opt = "force-pin-prompt", .required = false,       \
```

Comment:
Also here. Instead of reverting back a change from a previous commit, please update the original commit to not do the change at all.
The same may apply to more similar places.